src/main/java/org/primefaces/component/timeline/TimelineBase.java

```diff
@@ -212,6 +214,22 @@ public void setMaxHeight(Integer maxHeight) {
         getStateHelper().put(PropertyKeys.maxHeight, maxHeight);
     }
 
+    public Boolean getHorizontalScroll() {
+        return (Boolean) getStateHelper().eval(PropertyKeys.horizontalScroll, false);
+    }
+
+    public void setHorizontalScroll(Boolean horizontalScroll) {
+        getStateHelper().put(PropertyKeys.horizontalScroll, horizontalScroll);
+    }
+
+    public Boolean getVerticalScroll() {
```

Comment:
`isVerticalScroll` and `isHorizontalScroll` instead of `get`

Reply:
If "boolean" is used, "is" prefix should be used. If "Boolean" is used, "get" prefix should be used.
So its current actually

Reply:
Use `boolean`, not the wrapper 😉

Reply:
I replaced `Boolean`s with `boolean`s.